lens/src/main/resources/interpreter-setting.json

```diff
@@ -1,52 +1,51 @@
 [
-  {
-    "group": "lens",
-    "name": "lens",
-    "className": "org.apache.zeppelin.lens.LensInterpreter",
-    "properties": {
-      "zeppelin.lens.run.concurrent": {
-        "envName": null,
-        "propertyName": "zeppelin.lens.run.concurrent",
-        "defaultValue": "true",
-        "description": "Run concurrent Lens Sessions"
-      },
-      "zeppelin.lens.maxThreads": {
-        "envName": null,
-        "propertyName": "zeppelin.lens.maxThreads",
-        "defaultValue": "10",
-        "description": "If concurrency is true then how many threads?"
-      },
-      "zeppelin.lens.maxResults": {
-        "envName": null,
-        "propertyName": "zeppelin.lens.maxResults",
-        "defaultValue": "1000",
-        "description": "max number of rows to display"
-      },
-       "lens.server.base.url": {
-        "envName": null,
```

Comment:
@meenakshisekar I think @jongyoul meant not the whole file of indentation but only this block. As you can see, it has only 1 space while the other blocks are 2 spaces.